resources/language/resource.language.en_US/strings.po

```diff
@@ -777,3 +777,11 @@ msgstr ""
 #~ msgctxt "#32161"
```

Comment:
How do I generate the correct po files and add the missing translations?

Reply:
The base translation file is `en_gb` so you should only be editing that

Reply:
I'm noticing errors in git due to there being a `en_US` and `en_us` language folder for example.

```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'resources/language/resource.language.es_AR/strings.po'
  'resources/language/resource.language.es_ar/strings.po'
  'resources/language/resource.language.es_MX/strings.po'
  'resources/language/resource.language.es_mx/strings.po'
  'resources/language/resource.language.nb_NO/strings.po'
  'resources/language/resource.language.nb_no/strings.po'
  'resources/language/resource.language.pt_BR/strings.po'
  'resources/language/resource.language.pt_br/strings.po'

```

Reply:
that's probably a windows thing